app/controllers/application_controller.rb

```diff
@@ -1,4 +1,24 @@
 class ApplicationController < ActionController::API
   include Response
   include ExceptionHandler
+
+  around_action :with_current_request
+
+  private
+
+  def with_current_request
```

Comment:
@bzwei do you see no need to allow for an env variable to bypass_tenancy?

Reply:
no for now. We will follow your example if we feel it is needed for test/development